src/auxiliary.jl

```diff
@@ -69,7 +69,12 @@ end
 Return the coefficient of order `n::Int` of a `a::Taylor1` polynomial.
 """
 get_coeff(a::Taylor1, n::Int) = (@assert 0 ≤ n ≤ a.order+1;
-    return a.coeffs[n+1])
+    return a[n+1])
+
+getindex(a::Taylor1, n::Int) = a.coeffs[n]
+getindex(a::Taylor1, n::UnitRange) = a.coeffs[n]
```

Comment:
Don't we need also methods for `HomogeneousPolynomial` and `TaylorN`? This may be the cause that the tests are not passing.